src/genophenocorr/analysis/_commie.py

```diff
@@ -106,15 +106,22 @@ def __init__(self, cohort: Cohort,
                  missing_implies_excluded: bool = False,
                  include_sv: bool = False,
                  p_val_correction: typing.Optional[str] = None,
-                 min_perc_patients_w_hpo: typing.Union[float, int] = .1):
+                 min_perc_patients_w_hpo: typing.Union[float, int] = .1,
+                 recessive: bool = False):
         if not isinstance(cohort, Cohort):
             raise ValueError(f"cohort must be type Cohort but was type {type(cohort)}")
 
         self._logger = logging.getLogger(__name__)
+        handler = logging.FileHandler(f"{__name__}.log", mode='w')
```

Comment:
Logging must not be configured here. We must not make decisions on behalf of the user. The lines 115-118 must be removed.

Reply:
Removed